libpromises/mod_files.c

```diff
@@ -339,6 +339,7 @@ static const ConstraintSyntax CF_FILES_BODIES[] =
     ConstraintSyntaxNewBody("copy_from", &copy_from_body, "Criteria for copying file from a source", SYNTAX_STATUS_NORMAL),
     ConstraintSyntaxNewBool("create", "true/false whether to create non-existing file. Default value: false", SYNTAX_STATUS_NORMAL),
     ConstraintSyntaxNewBody("delete", &delete_body, "Criteria for deleting files", SYNTAX_STATUS_NORMAL),
+    ConstraintSyntaxNewString("content", CF_ANYSTRING, "Content for the file", SYNTAX_STATUS_NORMAL),
```

Comment:
I think this should say `Complete content the promised file should contain`

Reply:
Like: https://github.com/cfengine/documentation/pull/2327/commits/1e4ac3b20a675b7b15afc2173c42e447f030bf34

Reply:
Thanks. Fixed :+1: